EventsExpress/Controllers/AuthenticationController.cs

```diff
@@ -208,9 +208,16 @@ public async Task<IActionResult> EmailConfirm(string authLocalId, string token)
         [HttpPost("[action]")]
         public async Task<IActionResult> ChangePassword(ChangeViewModel model)
         {
-            await _authService.ChangePasswordAsync(model.OldPassword, model.NewPassword);
+            try
```

Comment:
Why do we need the try/catch here? The badrequest usually supposed to return something actionable for the user - in this case it will not.